src/linter/manifestJson/ManifestLinter.ts

```diff
@@ -89,16 +120,49 @@ export default class ManifestLinter {
 		}
 
 		// Detect deprecated view type in routing.config:
-		if (routing?.config && routing.config.viewType && deprecatedViewTypes.includes(routing.config.viewType)) {
+		const viewType = routing?.config?.viewType as string ?? routing?.config?.type;
+		if (viewType && deprecatedViewTypes.includes(viewType)) {
 			this.#reporter?.addMessage(MESSAGE.DEPRECATED_VIEW_TYPE, {
-				viewType: routing.config.viewType,
-			}, "/sap.ui5/routing/config/viewType");
+				viewType: viewType,
+			},
+			(routing?.config?.viewType ? "/sap.ui5/routing/config/viewType" : "/sap.ui5/routing/config/type"));
 		}
 
 		// Detect deprecations in routing.targets:
 		const targets = routing?.targets;
+		const oldToNewTargetPropsMap = {
+			viewName: "name",
+			viewId: "id",
+			viewLevel: "level",
+			viewPath: "path",
```

Comment:
Those properties may also be present in `routing.config`, where they should also be checked.

Reply:
Thanks!